CHANGELOG.md

```diff
@@ -2,6 +2,25 @@ faucet-pipeline-core version history
 ====================================
 
 
+v1.5.0
+------
+
+_2020-04-20_
+
+notable changes for end users:
+
+* configuration can now be asynchronous:
+
+  ```javascript
+  module.exports = new Promise(…);
+  ```
+
+*  updated dependencies - notably Browserslist, which now offers a command to
+   update the underlying feature-support matrix: `npx browserslist --update-db`
```

Comment:
I'm a little unsure about the wording here: Should caniuse.com's data set be considered a "feature-support matrix"?

More generally, do we want the burden of providing such PSAs? It seems in the spirit of faucet to summarize relevant news from around the ecosystem, but it also sets a precedent we might not wanna commit to for the future.

Background: https://cultofmartians.com/tasks/browserslist-update-db.html (auto-translation works well enough) - though [apparently it's a "rare feature"](https://github.com/browserslist/browserslist/pull/455#issuecomment-590107043), so I might be overestimating the significance here?

Reply:
I would personally not add this line to the changelog. This is not a command we provide. And I'm not sure it is super useful either 😉

Reply:
QED: You're a tech bro devoid of empathy.

Fortunately, so am I, thus I'm glad not to have to take on this (or any) responsibility.